pom.xml

```diff
@@ -116,9 +122,11 @@
 	</dependencyManagement>
 
 	<properties>
-		<openMRSVersion>1.9.9</openMRSVersion>
+		<openMRSVersion>2.0.0</openMRSVersion>
```

Comment:
Should this also be changed? https://github.com/openmrs/openmrs-module-patientmatching/blob/master/omod/src/main/resources/config.xml#L31

Reply:
Good catch! Added it